pkg/prometheus/statefulset.go

```diff
@@ -626,21 +650,17 @@ func makeStatefulSetSpec(p monitoringv1.Prometheus, c *Config, ruleConfigMapName
 		livenessFailureThreshold = 60
 	}
 
-	var livenessProbe *v1.Probe
-	var readinessProbe *v1.Probe
-	if !p.Spec.ListenLocal {
```

Comment:
Should we be removing this? I would maybe instead of considering adding a new field, wdyt? As you can't opt-out of this now if I understand correctly.

Reply:
Not sure I understand your question. This if basically moved up a bit to the place where we actually create those handlers. :slightly_smiling_face:

Reply:
discussed offline :)